.github/workflows/ci-osx.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
       - name: Set up env
         shell: "bash -l {0}"
         run: |
-          conda create -n env python==${{matrix.python-version}} wheel pip compilers
+          conda create -n env python=${{matrix.python-version}} wheel pip compilers 'clang>=12.0.1'
```

Comment:
Let's leave this for that PR to sort out 🙂

Reply:
You want to me revert the modification?

Reply:
Yes